docs/automated_testing.md

```diff
@@ -991,38 +993,162 @@ uses: suffolkLITLab/ALKiln@releases/v4
 
 When you want to update to a new version of the ALKiln, update that sha manually.
 
+### Set ALKiln's npm version
+
+This one requires prior technical knowledge. Feel free to ask us.
```

Comment:
I like the idea, but not sure if I like the presentation of these "technical knowledge" headers specifically, the one line disclaimer for such a small section, repeated through the page. 
Might have to wait until we start splitting up this documentation (which is already pretty long).

At the least, here's a suggestion for the phrasing:
```suggestion
This section requires prior technical knowledge about [npm](https://docs.npmjs.com/about-npm). Feel free to [ask us](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/#get-involved) about any questions you might have.
```

Reply:
If I recall correctly, it also needs knowledge of workflow files. I'm not sure where to point them to for that.

```suggestion
This section requires prior technical knowledge about [npm](https://docs.npmjs.com/about-npm) and GitHub workflow files. Feel free to [ask us](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/#get-involved) about any questions you might have.
```

Reply:
Good point, though I think we describe a lot about workflow files elsewhere in this page. But it's a little scattered.

```suggestion
This section requires prior technical knowledge about [npm](https://docs.npmjs.com/about-npm) and [GitHub workflow files](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions). Feel free to [ask us](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/#get-involved) about any questions you might have.
```